README.md

```diff
@@ -99,6 +99,7 @@ What it sets up
 [hub]: https://github.com/github/hub
 [ImageMagick]: http://www.imagemagick.org/
 [MySQL]: https://www.mysql.com/
+[n]: https://github.com/tj/n
```

Comment:
I've used both nvm and n and here are the basic diffs:
-  Shell manipulation
 - nvm uses trickery to modify the $PATH, and you have to source it's own `nvm.sh` before it works.
 - n basically `ln -sf` the prefered node version to a specific location, so things like `which node` or the like don't go nuts, and the only requirement is to add it's directory as `N_PREFIX`, and add `$N_PREFIX/bin` to  your `$PATH`
- Source vs binary
 - nvm downloads and compiles source versions of node if the binary is not available
 - n only downloads the binaries. However binaries are available from version 0.8.7+, so don't really see an issue